scripts/deis.sh

```diff
@@ -3,12 +3,24 @@
 deis_healthcheck() {
   wait-for-all-pods "deis"
   local successes
+  local failures
+  local max_attempts=10
   echo "Checking to see if the workflow has come up properly."
-  while [[ ${successes} -lt 10 ]]; do
+  while [[ ${successes} -lt "${max_attempts}" ]] && [[ ${failures} -lt "${max_attempts}" ]]; do
```

Comment:
@Joshua-Anderson good question -- I believe we'll still get where we want on the 'last' loop.  Consider if we are at 9 successes,  we'll enter the loop again, assuming we get another success, 9 gets bumped to 10 and then the 'final' success result occurs.